### PR TITLE
Update annotations for consistency

### PR DESCRIPTION
**Pull request by macintoshpie**
### Changes
- small improvements to the xsd documentation, primarily more consistent references.

### Additions
- added rake tasks for parsing and updating the documentation

### CODE REVIEW

**Comment by macintoshpie** [comment]
@nllong should we change this to merge into `prep-2.1-release`?

**Comment by nllong** [comment]
Merge into develop and I’ll pull up to the prep release branch.

File: BuildingSync.xsd [hunk 1]
```diff
@@ -6363,7 +6363,7 @@
                                         <xs:sequence>
                                           <xs:element name="CondenserPlantID" minOccurs="0" maxOccurs="unbounded">
                                             <xs:annotation>
-                                              <xs:documentation>ID numbers of the central condenser plants serving as the source for this cooling system.</xs:documentation>
+                                              <xs:documentation>ID number of the central CondensorPlant serving as the source for this cooling system.</xs:documentation>
```
Comment (nllong) [inline BuildingSync.xsd:6366]: Should be `CondenserPlant` @macintoshpie

File: BuildingSync.xsd [hunk 2]
```diff
@@ -7589,7 +7589,7 @@
                   <xs:sequence>
                     <xs:element name="CondenserPlantID" minOccurs="0" maxOccurs="unbounded">
                       <xs:annotation>
-                        <xs:documentation>ID numbers of the central condenser plants serving as the source for this cooling system.</xs:documentation>
+                        <xs:documentation>ID number of the central CondensorPlant serving as the source for this cooling system.</xs:documentation>
```
Comment (nllong) [inline BuildingSync.xsd:7592]: Condenser -- @macintoshpie

**Comment by nllong** [comment]
@macintoshpie Need to fix line 6366 too.

File: BuildingSync.xsd [hunk 3]
```diff
@@ -6363,7 +6363,7 @@
                                         <xs:sequence>
                                           <xs:element name="CondenserPlantID" minOccurs="0" maxOccurs="unbounded">
                                             <xs:annotation>
-                                              <xs:documentation>ID numbers of the central condenser plants serving as the source for this cooling system.</xs:documentation>
+                                              <xs:documentation>ID number of the central condenser plant serving as the source for this cooling system.</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:6366]: CondenserPlant

File: BuildingSync.xsd [hunk 4]
```diff
@@ -7589,7 +7589,7 @@
                   <xs:sequence>
                     <xs:element name="CondenserPlantID" minOccurs="0" maxOccurs="unbounded">
                       <xs:annotation>
-                        <xs:documentation>ID numbers of the central condenser plants serving as the source for this cooling system.</xs:documentation>
+                        <xs:documentation>ID number of the central condenser plant serving as the source for this cooling system.</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:7592]: CondenserPlant

File: BuildingSync.xsd [hunk 5]
```diff
@@ -9802,7 +9802,7 @@
                       <xs:sequence>
                         <xs:element name="CondenserPlantID" minOccurs="0" maxOccurs="unbounded">
                           <xs:annotation>
-                            <xs:documentation>ID numbers of the condenser plants serving as the source for this cooling system.</xs:documentation>
+                            <xs:documentation>ID numbers of the CondenserPlant serving as the source for this cooling system.</xs:documentation>
```
Comment (corymosiman12) [inline BuildingSync.xsd:9805]: ID number

**Comment by nllong** [comment]
@markborkum - do you want to opine on this before we merge?

**Comment by markborkum** [comment]
> @markborkum - do you want to opine on this before we merge?

@nllong The changes so far look good. Answering a question with a question: should we do a second pass to replace "(unit)" with "[unit]"?

**Comment by nllong** [comment]
I think we should wait until after the release to clean up the units. We have a process and it should be painless. I'm presently worried about the review time and other rabbit holes that we might get sucked into.

https://github.com/BuildingSync/schema/issues/210

**Pull request merged**